### DISCUSSION TITLE
Topic detection for other languages

### DISCUSSION THREAD

**Discussion by littlegreengeek** [deepgram/community#863] [Feature Requests]
Hi,
I would be very interested in topic detection for other languages in particular french. Is it on your roadmap, would you have a rough ETA  ?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@littlegreengeek I'll share this with the product team.
